specs/daito/Alibaba_OSS_Support.md

```diff
@@ -0,0 +1,329 @@
+# Integration of Alibaba Cloud as a backend in OpenSDS/Multi-cloud
+**Author(s)**:  [Click2Cloud Technology Services India Pvt. Ltd.](https://github.com/Click2Cloud)
+### Summary
+Using this feature, user will be able to do object management mechanism and migration for Alibaba Cloud in multi-cloud .
+### Motivation
+In existing OpenSDS Gelato project, we don't have any option using which user can manage object management mechanism and migration for Alibaba Cloud. So we adding  alibaba as backend for Object management and migration.
+### Goals
+
+* Add support of Alibaba OSS service in OpenSDS Gelato Project.
+
+### Design Details
+![picture](Alibaba_backend.png)
+
+                    Figure: Architecture of Integration of Alibaba Cloud in OpenSDS/Multi-cloud
+
+By adding Alibaba OSS as adapter  in OpenSDS will help users to perform all bucket and object related operations as follow:
+* Add Backend for Alibaba OSS (`alibaba-oss` type)
+* Create OpenSDS Bucket on Alibaba OSS
+* Delete Bucket from Alibaba OSS
+* SSE supported
+* Upload Object on Alibaba OSS (PUT Object)  
```

Comment:
This part is analysed? still issues there?

Reply:
No we have tested with latest Elba release too, it works fine. Thanks